NEWS.md

```diff
@@ -11,6 +11,14 @@
 
 * Failure locations are now formated as R error locations.
 
+* The [Catch](https://github.com/philsquared/Catch) unit
+  testing library can now be used for unit testing of C++
+  files. You can place C++ files within the `tests/testthat/cpp`
```

Comment:
I was worried about potential collisions with other filters that e.g. just look for `test-.*` -- can I expect that all pre-existing filters / commands are properly scoped (and so only pick up `R` test files in that folder)?